rewrite-core/src/main/java/org/openrewrite/config/YamlResourceLoader.java

```diff
@@ -167,17 +169,19 @@ public YamlResourceLoader(InputStream yamlInput, URI source, Properties properti
 
     private Collection<Map<String, Object>> loadResources(ResourceType resourceType) {
```

Comment:
Can we add a test to validate that this do not happen again? It is fine if we want to include it using a different PR